### ISSUE TITLE
About Loss Function

### ISSUE THREAD

**Issue by rosefun** [blackecho/Deep-Learning-TensorFlow#65]
In `rbm.py` , I find `from yadlt.core import Layers, Loss`. However, the folder does not include Loss.py.

Although,I can run the project without error. But ,how does it do?

RBM `_init_` include `self.loss = Loss(self.loss_func)`,so how does it run?

Hopefully for reply.

Thanks and best wishes!

**Comment by rosefun** [comment]
I find that there are a Class Object called Loss.
